assets/src/settings-page/analytics.js

```diff
@@ -17,6 +18,38 @@ import { Button, TextControl, PanelRow, BaseControl } from '@wordpress/component
  */
 import { AMPDrawer } from '../components/amp-drawer';
 import { Options } from '../components/options-context-provider';
+import { AMPNotice, NOTICE_SIZE_SMALL } from '../components/amp-notice';
+
+const GOOGLE_ANALYTICS_VENDOR = 'googleanalytics';
```

Comment:
Site Kit technically uses `gtag` vendor, which is also Google Analytics (the recommended way over what's `googleanalytics` in AMP). So I think we should display this recommendation for both `googleanalytics` and `gtag`.

Reply:
How about this?

![image](https://user-images.githubusercontent.com/134745/89813187-1e193580-daf6-11ea-9d48-62e0966b8e5b.png)

See 66b807191.

Reply:
LGTM, thanks!